openslides/mediafiles/static/js/mediafiles/site.js

```diff
@@ -111,6 +126,111 @@ angular.module('OpenSlidesApp.mediafiles.site', ['ngFileUpload', 'OpenSlidesApp.
         $scope.delete = function (mediafile) {
             Mediafile.destroy(mediafile.id);
         };
+
+        // ** PDF presentation functions **/
+        // show document on projector
+        $scope.showPdf = function (mediafile) {
+            var postUrl,
+                data;
+            if($scope.presentedMediafiles.length > 0) {
+                // update first mediafile, at the moment there should not be more
```

Comment:
yes, it's good enough for 2.0b2. we will improve the whole pdf presentation performance later.